actions/locations.js

```diff
@@ -134,13 +159,58 @@ export function createLocation(options) {
         return;
       }
 
-      const { created: location, saved } = await apis.createLocation(regionKey, locationData, key);
+      const { created: location, saved } = await apiCall(regionKey, key, localLocation);
```

Comment:
⚠️ It looks like this call switched from using `locationData` to `localLocation`. I cannot tell is that is a bad thing. It sure is different.

Reply:
This is done on purpose. The data is updated or created in the local database first and the result of that is then used to send to the server since the `localLocation` object contains info like the location version number, where as the `locationData` object does not.

Reply:
How does versioning/changes work if it is only local?

Reply:
Could you be a little more specific? When sending to the remote we do store the version info. It's just that the `locationData` object here doesn't necessarily include that information.

Reply:
I was expecting the version number to be remote since we need to make sure data is synced correctly in firebase.

Your comment cleared it up a bit. So we do send it, just not necessarily initially.